vapi/simulator/simulator.go

```diff
@@ -475,6 +479,52 @@ func (s *handler) category(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *handler) shutdown(w http.ResponseWriter, r *http.Request) {
+	done := func(delay int) {
+		go time.AfterFunc(time.Duration(delay)*time.Minute, func() {
```

Comment:
Nice!

Reply:
+1

My proposal would be to slightly change this for better readability to:

```go
delayShutdown := func(delay int) {
		time.Sleep(delay * time.Minute)
		s.Lock()
		s.ShutdownStatus.ShutDownTime = ""
		s.ShutdownStatus.Action = ""
		s.ShutdownStatus.Reason = ""
		s.Unlock()
	}

// in the corresponding case block use:
go delayShutdown()
```

Reply:
Updated to 
```
	delayShutdown := func(delay int) {
		go time.AfterFunc(time.Duration(delay)*time.Minute, func() {
			s.Lock()
			s.Shutdown.ShutDownTime = ""
			s.Shutdown.Action = ""
			s.Shutdown.Reason = ""
			s.Unlock()
		})
	}
```